ui/src/jobs/components/RegionsList.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+
+import InlineList from '../../common/components/InlineList';
+
+class RegionsList extends Component {
+  render() {
+    const { regions } = this.props;
+    return <InlineList items={regions} renderItem={item => item} />;
```

Comment:
this is already default `renderItem`

Reply:
Actually this is default extractKey, renderItem is required